CSharp_Course_Project/Program.cs

```diff
@@ -6,10 +6,20 @@ class Program
     {
         static void Main(string[] args)
         {
-            squareCircle();
-            Average();
-            Volume();
-            Area();
+            //squareCircle();
```

Comment:
Don't forget that methods, classes, properties, events etc. should have names that match Pascal casing.